### ISSUE TITLE
Use Support navigation drawer instead of third party

### ISSUE THREAD

**Issue by emartynov** [Glucosio/glucosio-android#392]
I've removed date/time pickers and FAB. But removing drawer requires a bit more time.

**Comment by paolorotolo** [comment]
Drawer library isn't deprecated and is well supported. We can keep it IMHO.

**Comment by AndyScherzinger** [comment]
Both variants seem fine to me. I did the support lib drawer implementation for Nextcloud so that is doable while the lib offers a lot of on top functionality. So both have their upsides.

**Comment by emartynov** [comment]
Up to you. As for me, less third-party dependencies is better.

**Comment by AndyScherzinger** [comment]
I’d leave the decision to @paolorotolo

**Comment by emartynov** [comment]
I've asked Mike about it https://github.com/mikepenz/MaterialDrawer/pull/2171

**Comment by paolorotolo** [comment]
Yes, I like the idea to reduce the amount of third-party libraries in the app but I also don't want to reinvent the wheel.
I've seen Mike answer and **for now** our implementation of the drawer is pretty standard and could be replaced by the Support Library without issues.

On another side, we might need to introduce login very soon (to enable @chrispecoraro's API) and add Account support to the drawer, and this is easily achievable with Mike Penz's library.

**Comment by emartynov** [comment]
I'm not sure about login for the user. It doesn't give much additional functionality and only will be added by developers to achieve unique ID for uploaded data. That might be resolved by just GUID generation.

Unless we have something on the roadmap that will also use login.

**Comment by piotrek1543** [comment]
Login? I feel that someone here has a vision of a user that wants to create an account and keep his/her data on (remote) server and he wants to then use A/B tests and analytics to get to know how user uses the app - in one word - Firebase Services :-)

**Comment by paolorotolo** [comment]
No, as @emartynov said we only need login to get an ID for our API.
@bkerensa can say more about our vision, one of the key principles of our project is to keep users in complete control of their data, for this I wouldn't consider for now the option to upload data to third-party servers.

Unless, if we really want to explore the possibility of sync between devices, we can add login to a personal NextCloud server or something like that. But again, we need to discuss this properly and we're going a little OT here.

**Comment by ghost** [comment]
@emartynov  @paolorotolo guys, could you update [projects](https://github.com/Glucosio/glucosio-android/projects) kanban? It looks old and has some issues are closed.
